src/Resources/contao/classes/BackendUser.php

```diff
@@ -154,39 +158,6 @@ public function __get($strKey)
 		return parent::__get($strKey);
 	}
 
-
-	/**
-	 * Redirect to the login screen if authentication fails
-	 *
-	 * @return boolean True if the user could be authenticated
-	 */
-	public function authenticate()
```

Comment:
Sadly you must keep this method as it is part of the API and must continue to work as it does currently.